frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -82,7 +82,7 @@
                     <template v-if="creatingNew" #appended-description>
                         <p v-if="hasValidName" class="instance-name-confirmation">
                             <CheckCircleIcon class="ff-btn--icon" />
-                            <span>Your instance will be created as "<i>{{ instanceName }}</i>".</span>
+                            <span>Your instance will be created as "<i>{{ instanceName.toLowerCase() }}</i>".</span>
```

Comment:
If we're accepting instance names with uppercase, is there any point in hiding it from users?

Reply:
```suggestion
                            <span>Your instance hostname will be "<i>{{ instanceName.toLowerCase() }}</i>".</span>
```

Reply:
How about with that change to the label

Reply:
makes more sense!